go/streamlog/streamlog_flaky_test.go

```diff
@@ -213,7 +213,7 @@ func TestFile(t *testing.T) {
 	logger.Send(&logMessage{"test 2"})
 
 	// Allow time for propagation
-	time.Sleep(10 * time.Millisecond)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Curious, I assume 10 ms was too slow? These sleeps in CI have always been a bit of a headache.

Reply:
Yeah, I had seen it flake.